### PR TITLE
Fix error when devicesupport is missing

### PR DESCRIPTION
**Pull request by diskshima**
The command was failing when the devicesupport directory (e.g. watchOS, tvOS, etc) was missing.

### CODE REVIEW

File: lib/devicesupport/deviceSupports.go [hunk 1]
```diff
@@ -18,12 +19,21 @@ func (v deviceSupports) path() string {
 }
 
 func (v deviceSupports) versions() ([]deviceSupport, error) {
-	files, err := ioutil.ReadDir(v.path())
+	path := v.path()
+
+	var list []deviceSupport
+
+	exist, err := exists(path)
```
Comment (yutailang0119) [inline lib/devicesupport/deviceSupports.go]: Its name should start with `is`, because variable type is bool.

Comment (yutailang0119) [reply lib/devicesupport/deviceSupports.go]: Since this err is not caught, I think it is able to omit.

**Review by yutailang0119 — COMMENTED** [review]
@diskshima Thanks for fixing the problem I couldn't find!
Just left a couple of small comments. 😃

**Comment by diskshima** [comment]
@yutailang0119 Thanks the review. Fixed and pushed.

**Review by yutailang0119 — APPROVED** [review]
@diskshima It looks good. Thanks!

**Pull request merged**